README.md

```diff
@@ -123,152 +125,244 @@ Congratulations! You can now send remote push notifications to your device from
+}
+```
+	
+##### 2. Deep Linking
+
+* Handling Links from Push Notifications
+	
+Push notifications and action buttons may have `openUrl` actions attached to them. When a URL is specified, the SDK first calls `urlDelegate` specified in your `IterableConfig` object. You can use this delegate to handle `openUrl` actions the same way as you handle normal deep links. If the delegate is not set or if it returns `false` (the default), the SDK will open Safari with that URL. If you want to navigate to a UIViewController on receiving a deep link, you should do so in the `urlDelegate`. 
+	
+In the code below, `DeepLinkHandler` is a custom handler which is reponsible for deep link navigation. You have to provide implementation for deep link navigation. Please see [sample application](https://github.com/Iterable/swift-sdk/blob/master/sample-apps/swift-sample-app?raw=true) for a reference implementation.
+	
+**Swift**
+	
+```swift
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+	...
+	// Initialize Iterable API
+	let config = IterableConfig()
+	...
+	config.urlDelegate = self
+	IterableAPI.initialize(apiKey: apiKey, launchOptions:launchOptions, config: config)
+	...
+}
+
+// Iterable URL Delegate. It will be called when you receive 
+// an `openUrl` event from push notification.
+func handle(iterableURL url: URL, inContext context: IterableActionContext) -> Bool {
+	return DeeplinkHandler.handle(url: url)
+}
+```
+	
+**Objective-C**
+	
+```objective-c
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+	...
+	// Initialize Iterable SDK
+	IterableConfig *config = [[IterableConfig alloc] init];
+	...
+	config.urlDelegate = self;
+	[IterableAPI initializeWithApiKey:@"YOUR API KEY" launchOptions:launchOptions config:config];
+	...
+}
+	
+- (BOOL)handleIterableURL:(NSURL *)url context:(IterableActionContext *)context {
+	// Assuming you have a DeeplinkHandler class that handles all deep link URLs and navigates to the right place in the app
+	return [DeeplinkHandler handleUrl:url];
+}
+```
+		
+* Handling Email Links
+	
+For Universal Links to work with link rewriting in emails, you need to set up apple-app-site-association file in the Iterable project. More instructions here: [Setting up iOS Universal Links](https://support.iterable.com/hc/en-us/articles/115000440206-Setting-up-iOS-Universal-Links).
+
+When an email link is clicked your `UIApplicationDelegate`'s [application:continueUserActivity:restorationHandler:](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1623072-application?language=swift) method is called. If you already have an Iterable `urlDelegate` defined (see *Handling Links from Push Notifications* section above), the same handler can be used for email deep links by calling `handleUniversalLink:`.
```

Comment:
- is clicked,
- Actually link up to the prior section with an anchor link.
- Swift references instead of Objective-C references?

Reply:
Not sure I understand "link up to prior section...". Also, the reference here is Swift not ObjC.